sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/job_service.py

```diff
@@ -57,18 +57,6 @@ def __init__(
         self._validate_nodes()
         self._validate_job_service_type_name()
 
-    def _to_rest_object(self) -> RestJobService:
-        return RestJobService(
-            endpoint=self.endpoint,
-            job_service_type=JobServiceTypeNames.ENTITY_TO_REST.get(self.job_service_type, None)
-            if self.job_service_type
-            else None,
-            nodes=AllNodes() if self.nodes else None,
-            status=self.status,
-            port=self.port,
-            properties=self.properties,
-        )
-
     def _validate_nodes(self):
         if not self.nodes in ["all", None]:
             msg = f"nodes should be either 'all' or None, but received '{self.nodes}'."
```

Comment:
I see this here. But will there be a service that runs on only a couple of nodes?

Reply:
The services will either run in all or none as per the discussion with IJ team. Will double check with Srujan.

Reply:
We only support all or none as of today.

Reply:
@TonyJ1 @srsaggam, thank you for the context. Let's also consider what the API will be once we support a service running on a set of nodes so that the API is future-proof.